astrodynamics-fall-2018/astrodynamicsFall2018PDD.tex

```diff
@@ -0,0 +1,372 @@
+            Design mounting for selected LED
+            Determine placement of modifications
+            Install system at observatory & 2 weeks \\
+            7 & Systems testing & 2 weeks \\
+            8 & Generate documentation and delivery to SPEX & 1 week \\
+            % LaTeX doesn't really like multi-line cell contents. Try to keep the text in each cell concise!
+            \bottomrule
+        \end{tabularx}
+    \label{tab:timeline2}
+    \end{table}
+
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+  Leadership with the prerequisite skills has been determined.
+  It would be beneficial for some students to already have basic engineering and hand tool skills.
+  Knowledge of Arduino code and basic electrical circuits is also beneficial.
+  These skills do not need to be present in all group members as these projects are an opportunity for new members to learn these skills.
+  For the roll off motor repair, someone with some motor knowledge would be crucial.
+
+\subsection{Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+  Primary funding for these projects will come from the Observatory itself.
+  If extra funds are needed, RIT SPEX Astrodynamics may apply for SG funding due to their newly developed SG club status.
+  \autoref{tab:costEstimate} outlines the estimated costs of the projects.
+  The costs have been overestimated for planning purposes.
+
+  \begin{table}[hb!]
+    % this table is too wide for the two-column format, so we let it expand across both columns
+    % we haven't told LaTeX where to put this so it'll find the best place.
+        \caption{Estimated Budget}
+        \centering
+        \begin{tabularx}{\linewidth}{lXl}
+            % READ THIS!! https://www.inf.ethz.ch/personal/markusp/teaching/guides/guide-tables.pdf
+            \toprule % line on top external edge of table
+            % Separate cells in a row with &, move to the next row with \\
+            Project & Cost \\
+            \midrule % line separating two internal rows
+            Roll Off Motor Repair & 500 \\
+            Motor Latch Arm & 100 \\
+            12" Telescope Modifications & 100 \\
+            % LaTeX doesn't really like multi-line cell contents. Try to keep the text in each cell concise!
+            \bottomrule
+        \end{tabularx}
+    \label{tab:costEstimate}
+    \end{table}
+
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+  Direct support from Dr. Richmond and Dr. Connelly is expected.
```

Comment:
What kind of direct support is expected? Engineering design work? Mentorship? Physical fabrication or installation help?